scenarios/hitech95/mail-generic-bf.yaml

```diff
@@ -0,0 +1,30 @@
+# Global brouteforce
+type: leaky
+#debug: true
+name: crowdsecurity/email-generic-bf
```

Comment:
Please change `crowdsecurity` to `hitech95`

Reply:
Should be done 😃